Assets/SEETests/TestConfigIO.cs

```diff
@@ -685,8 +686,8 @@ private static void AreEqual(Constraint expected, Constraint actual)
         private static void SEEEvolutionCityAttributesAreEqual(SEECityEvolution expected, SEECityEvolution actual)
         {
             AbstractSEECityAttributesAreEqual(expected, actual);
-            AreEqual(expected.GXLDirectory, actual.GXLDirectory);
-            Assert.AreEqual(expected.MaxRevisionsToLoad, actual.MaxRevisionsToLoad);
+            TestGraphProviderIO.AreEqual(expected.DataProvider, actual.DataProvider);
+            //Assert.AreEqual(expected.MaxRevisionsToLoad, actual.MaxRevisionsToLoad);
```

Comment:
Why was this commented out? Can we remove it?

Reply:
Code commented out can be removed. The attribute does no longer exist in this class. I fixed it.